ERROR - 2025-07-24 10:59:33 --> mysqli_sql_exception: Unknown column 'latitude' in 'field list' in E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `at...', 0)
#1 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `at...')
#2 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `at...')
#3 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `at...', Array, false)
#4 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 E:\laragon\www\apici\vendor\codeigniter4\framework\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 E:\laragon\www\apici\app\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert(Array)
#8 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
#9 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
#10 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 E:\laragon\www\apici\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 E:\laragon\www\apici\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('E:\\laragon\\www\\...')
#15 {main}
CRITICAL - 2025-07-24 10:59:33 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'latitude' in 'field list'
[Method: POST, Route: api/attendance/checkin]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (:user_id:, :latitude:, :longitude:, :address:, :shift:, :work_location_type:, :time_in:, :attendance_date:, :photo_in:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
CRITICAL - 2025-07-24 10:59:33 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'latitude' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.343192\', \'106.7887855\', \'Jl. Damai No.100, Gandul, Kecamatan Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'10:59:33\', \'2025-07-24\', \'1753329573_1f4ec5262bbcb04c25b8.jpg\', \'2025-07-24 10:59:33\', \'2025-07-24 10:59:33\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.343192\', \'106.7887855\', \'Jl. Damai No.100, Gandul, Kecamatan Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'10:59:33\', \'2025-07-24\', \'1753329573_1f4ec5262bbcb04c25b8.jpg\', \'2025-07-24 10:59:33\', \'2025-07-24 10:59:33\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (:user_id:, :latitude:, :longitude:, :address:, :shift:, :work_location_type:, :time_in:, :attendance_date:, :photo_in:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
CRITICAL - 2025-07-24 10:59:33 --> [Caused by] mysqli_sql_exception: Unknown column 'latitude' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.343192\', \'106.7887855\', \'Jl. Damai No.100, Gandul, Kecamatan Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'10:59:33\', \'2025-07-24\', \'1753329573_1f4ec5262bbcb04c25b8.jpg\', \'2025-07-24 10:59:33\', \'2025-07-24 10:59:33\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.343192\', \'106.7887855\', \'Jl. Damai No.100, Gandul, Kecamatan Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'10:59:33\', \'2025-07-24\', \'1753329573_1f4ec5262bbcb04c25b8.jpg\', \'2025-07-24 10:59:33\', \'2025-07-24 10:59:33\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.343192\', \'106.7887855\', \'Jl. Damai No.100, Gandul, Kecamatan Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'10:59:33\', \'2025-07-24\', \'1753329573_1f4ec5262bbcb04c25b8.jpg\', \'2025-07-24 10:59:33\', \'2025-07-24 10:59:33\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (:user_id:, :latitude:, :longitude:, :address:, :shift:, :work_location_type:, :time_in:, :attendance_date:, :photo_in:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
ERROR - 2025-07-24 10:59:36 --> mysqli_sql_exception: Unknown column 'latitude' in 'field list' in E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `at...', 0)
#1 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `at...')
#2 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `at...')
#3 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `at...', Array, false)
#4 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 E:\laragon\www\apici\vendor\codeigniter4\framework\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 E:\laragon\www\apici\app\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert(Array)
#8 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
#9 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
#10 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 E:\laragon\www\apici\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 E:\laragon\www\apici\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('E:\\laragon\\www\\...')
#15 {main}
CRITICAL - 2025-07-24 10:59:36 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'latitude' in 'field list'
[Method: POST, Route: api/attendance/checkin]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (:user_id:, :latitude:, :longitude:, :address:, :shift:, :work_location_type:, :time_in:, :attendance_date:, :photo_in:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
CRITICAL - 2025-07-24 10:59:36 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'latitude' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.343192\', \'106.7887855\', \'Jl. Damai No.100, Gandul, Kecamatan Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'10:59:36\', \'2025-07-24\', \'1753329576_878398e9aa2c82ef15fb.jpg\', \'2025-07-24 10:59:36\', \'2025-07-24 10:59:36\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.343192\', \'106.7887855\', \'Jl. Damai No.100, Gandul, Kecamatan Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'10:59:36\', \'2025-07-24\', \'1753329576_878398e9aa2c82ef15fb.jpg\', \'2025-07-24 10:59:36\', \'2025-07-24 10:59:36\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (:user_id:, :latitude:, :longitude:, :address:, :shift:, :work_location_type:, :time_in:, :attendance_date:, :photo_in:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
CRITICAL - 2025-07-24 10:59:36 --> [Caused by] mysqli_sql_exception: Unknown column 'latitude' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.343192\', \'106.7887855\', \'Jl. Damai No.100, Gandul, Kecamatan Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'10:59:36\', \'2025-07-24\', \'1753329576_878398e9aa2c82ef15fb.jpg\', \'2025-07-24 10:59:36\', \'2025-07-24 10:59:36\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.343192\', \'106.7887855\', \'Jl. Damai No.100, Gandul, Kecamatan Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'10:59:36\', \'2025-07-24\', \'1753329576_878398e9aa2c82ef15fb.jpg\', \'2025-07-24 10:59:36\', \'2025-07-24 10:59:36\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.343192\', \'106.7887855\', \'Jl. Damai No.100, Gandul, Kecamatan Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'10:59:36\', \'2025-07-24\', \'1753329576_878398e9aa2c82ef15fb.jpg\', \'2025-07-24 10:59:36\', \'2025-07-24 10:59:36\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (:user_id:, :latitude:, :longitude:, :address:, :shift:, :work_location_type:, :time_in:, :attendance_date:, :photo_in:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
ERROR - 2025-07-24 11:00:02 --> mysqli_sql_exception: Unknown column 'latitude' in 'field list' in E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `at...', 0)
#1 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `at...')
#2 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `at...')
#3 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `at...', Array, false)
#4 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 E:\laragon\www\apici\vendor\codeigniter4\framework\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 E:\laragon\www\apici\app\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert(Array)
#8 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
#9 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
#10 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 E:\laragon\www\apici\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 E:\laragon\www\apici\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('E:\\laragon\\www\\...')
#15 {main}
CRITICAL - 2025-07-24 11:00:02 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'latitude' in 'field list'
[Method: POST, Route: api/attendance/checkin]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (:user_id:, :latitude:, :longitude:, :address:, :shift:, :work_location_type:, :time_in:, :attendance_date:, :photo_in:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
CRITICAL - 2025-07-24 11:00:02 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'latitude' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3432006\', \'106.7887833\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'11:00:02\', \'2025-07-24\', \'1753329602_3ddaf10b7cf15e6c9dea.jpg\', \'2025-07-24 11:00:02\', \'2025-07-24 11:00:02\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3432006\', \'106.7887833\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'11:00:02\', \'2025-07-24\', \'1753329602_3ddaf10b7cf15e6c9dea.jpg\', \'2025-07-24 11:00:02\', \'2025-07-24 11:00:02\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (:user_id:, :latitude:, :longitude:, :address:, :shift:, :work_location_type:, :time_in:, :attendance_date:, :photo_in:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
CRITICAL - 2025-07-24 11:00:02 --> [Caused by] mysqli_sql_exception: Unknown column 'latitude' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3432006\', \'106.7887833\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'11:00:02\', \'2025-07-24\', \'1753329602_3ddaf10b7cf15e6c9dea.jpg\', \'2025-07-24 11:00:02\', \'2025-07-24 11:00:02\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3432006\', \'106.7887833\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'11:00:02\', \'2025-07-24\', \'1753329602_3ddaf10b7cf15e6c9dea.jpg\', \'2025-07-24 11:00:02\', \'2025-07-24 11:00:02\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3432006\', \'106.7887833\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'11:00:02\', \'2025-07-24\', \'1753329602_3ddaf10b7cf15e6c9dea.jpg\', \'2025-07-24 11:00:02\', \'2025-07-24 11:00:02\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (:user_id:, :latitude:, :longitude:, :address:, :shift:, :work_location_type:, :time_in:, :attendance_date:, :photo_in:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
ERROR - 2025-07-24 11:04:30 --> mysqli_sql_exception: Unknown column 'latitude' in 'field list' in E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `at...', 0)
#1 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `at...')
#2 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `at...')
#3 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `at...', Array, false)
#4 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 E:\laragon\www\apici\vendor\codeigniter4\framework\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 E:\laragon\www\apici\app\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert(Array)
#8 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
#9 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
#10 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 E:\laragon\www\apici\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 E:\laragon\www\apici\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('E:\\laragon\\www\\...')
#15 {main}
CRITICAL - 2025-07-24 11:04:30 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'latitude' in 'field list'
[Method: POST, Route: api/attendance/checkin]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (:user_id:, :latitude:, :longitude:, :address:, :shift:, :work_location_type:, :time_in:, :attendance_date:, :photo_in:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
CRITICAL - 2025-07-24 11:04:30 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'latitude' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3431922\', \'106.7887769\', \'Jl. Damai No.100, Gandul, Kecamatan Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'11:04:30\', \'2025-07-24\', \'1753329870_df0b95df08bfb39aabb4.jpg\', \'2025-07-24 11:04:30\', \'2025-07-24 11:04:30\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3431922\', \'106.7887769\', \'Jl. Damai No.100, Gandul, Kecamatan Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'11:04:30\', \'2025-07-24\', \'1753329870_df0b95df08bfb39aabb4.jpg\', \'2025-07-24 11:04:30\', \'2025-07-24 11:04:30\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (:user_id:, :latitude:, :longitude:, :address:, :shift:, :work_location_type:, :time_in:, :attendance_date:, :photo_in:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
CRITICAL - 2025-07-24 11:04:30 --> [Caused by] mysqli_sql_exception: Unknown column 'latitude' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3431922\', \'106.7887769\', \'Jl. Damai No.100, Gandul, Kecamatan Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'11:04:30\', \'2025-07-24\', \'1753329870_df0b95df08bfb39aabb4.jpg\', \'2025-07-24 11:04:30\', \'2025-07-24 11:04:30\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3431922\', \'106.7887769\', \'Jl. Damai No.100, Gandul, Kecamatan Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'11:04:30\', \'2025-07-24\', \'1753329870_df0b95df08bfb39aabb4.jpg\', \'2025-07-24 11:04:30\', \'2025-07-24 11:04:30\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3431922\', \'106.7887769\', \'Jl. Damai No.100, Gandul, Kecamatan Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'11:04:30\', \'2025-07-24\', \'1753329870_df0b95df08bfb39aabb4.jpg\', \'2025-07-24 11:04:30\', \'2025-07-24 11:04:30\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (:user_id:, :latitude:, :longitude:, :address:, :shift:, :work_location_type:, :time_in:, :attendance_date:, :photo_in:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
ERROR - 2025-07-24 11:08:05 --> mysqli_sql_exception: Unknown column 'latitude' in 'field list' in E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `at...', 0)
#1 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `at...')
#2 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `at...')
#3 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `at...', Array, false)
#4 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 E:\laragon\www\apici\vendor\codeigniter4\framework\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 E:\laragon\www\apici\app\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert(Array)
#8 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
#9 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
#10 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 E:\laragon\www\apici\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 E:\laragon\www\apici\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('E:\\laragon\\www\\...')
#15 {main}
CRITICAL - 2025-07-24 11:08:05 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'latitude' in 'field list'
[Method: POST, Route: api/attendance/checkin]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (:user_id:, :latitude:, :longitude:, :address:, :shift:, :work_location_type:, :time_in:, :attendance_date:, :photo_in:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
CRITICAL - 2025-07-24 11:08:05 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'latitude' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3431954\', \'106.7887923\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'11:08:05\', \'2025-07-24\', \'1753330085_dd58ec05222b094aec06.jpg\', \'2025-07-24 11:08:05\', \'2025-07-24 11:08:05\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3431954\', \'106.7887923\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'11:08:05\', \'2025-07-24\', \'1753330085_dd58ec05222b094aec06.jpg\', \'2025-07-24 11:08:05\', \'2025-07-24 11:08:05\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (:user_id:, :latitude:, :longitude:, :address:, :shift:, :work_location_type:, :time_in:, :attendance_date:, :photo_in:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
CRITICAL - 2025-07-24 11:08:05 --> [Caused by] mysqli_sql_exception: Unknown column 'latitude' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3431954\', \'106.7887923\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'11:08:05\', \'2025-07-24\', \'1753330085_dd58ec05222b094aec06.jpg\', \'2025-07-24 11:08:05\', \'2025-07-24 11:08:05\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3431954\', \'106.7887923\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'11:08:05\', \'2025-07-24\', \'1753330085_dd58ec05222b094aec06.jpg\', \'2025-07-24 11:08:05\', \'2025-07-24 11:08:05\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3431954\', \'106.7887923\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'11:08:05\', \'2025-07-24\', \'1753330085_dd58ec05222b094aec06.jpg\', \'2025-07-24 11:08:05\', \'2025-07-24 11:08:05\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (:user_id:, :latitude:, :longitude:, :address:, :shift:, :work_location_type:, :time_in:, :attendance_date:, :photo_in:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
ERROR - 2025-07-24 11:21:10 --> mysqli_sql_exception: Unknown column 'latitude' in 'field list' in E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `at...', 0)
#1 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `at...')
#2 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `at...')
#3 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `at...', Array, false)
#4 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 E:\laragon\www\apici\vendor\codeigniter4\framework\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 E:\laragon\www\apici\app\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert(Array)
#8 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
#9 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
#10 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 E:\laragon\www\apici\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 E:\laragon\www\apici\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('E:\\laragon\\www\\...')
#15 {main}
CRITICAL - 2025-07-24 11:21:10 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'latitude' in 'field list'
[Method: POST, Route: api/attendance/checkin]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (:user_id:, :latitude:, :longitude:, :address:, :shift:, :work_location_type:, :time_in:, :attendance_date:, :photo_in:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
CRITICAL - 2025-07-24 11:21:10 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'latitude' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3431993\', \'106.7887784\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'11:21:10\', \'2025-07-24\', \'1753330870_6c0a9ab5f4c075d9fa67.jpg\', \'2025-07-24 11:21:10\', \'2025-07-24 11:21:10\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3431993\', \'106.7887784\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'11:21:10\', \'2025-07-24\', \'1753330870_6c0a9ab5f4c075d9fa67.jpg\', \'2025-07-24 11:21:10\', \'2025-07-24 11:21:10\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (:user_id:, :latitude:, :longitude:, :address:, :shift:, :work_location_type:, :time_in:, :attendance_date:, :photo_in:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
CRITICAL - 2025-07-24 11:21:10 --> [Caused by] mysqli_sql_exception: Unknown column 'latitude' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3431993\', \'106.7887784\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'11:21:10\', \'2025-07-24\', \'1753330870_6c0a9ab5f4c075d9fa67.jpg\', \'2025-07-24 11:21:10\', \'2025-07-24 11:21:10\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3431993\', \'106.7887784\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'11:21:10\', \'2025-07-24\', \'1753330870_6c0a9ab5f4c075d9fa67.jpg\', \'2025-07-24 11:21:10\', \'2025-07-24 11:21:10\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3431993\', \'106.7887784\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'11:21:10\', \'2025-07-24\', \'1753330870_6c0a9ab5f4c075d9fa67.jpg\', \'2025-07-24 11:21:10\', \'2025-07-24 11:21:10\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (:user_id:, :latitude:, :longitude:, :address:, :shift:, :work_location_type:, :time_in:, :attendance_date:, :photo_in:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
ERROR - 2025-07-24 11:41:34 --> mysqli_sql_exception: Unknown column 'latitude' in 'field list' in E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `at...', 0)
#1 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `at...')
#2 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `at...')
#3 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `at...', Array, false)
#4 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 E:\laragon\www\apici\vendor\codeigniter4\framework\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 E:\laragon\www\apici\app\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert(Array)
#8 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
#9 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
#10 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 E:\laragon\www\apici\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 E:\laragon\www\apici\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('E:\\laragon\\www\\...')
#15 {main}
CRITICAL - 2025-07-24 11:41:34 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'latitude' in 'field list'
[Method: POST, Route: api/attendance/checkin]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (:user_id:, :latitude:, :longitude:, :address:, :shift:, :work_location_type:, :time_in:, :attendance_date:, :photo_in:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
CRITICAL - 2025-07-24 11:41:34 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'latitude' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3431993\', \'106.7887784\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'11:41:34\', \'2025-07-24\', \'1753332093_52ec45cb5f721a1172bd.jpg\', \'2025-07-24 11:41:34\', \'2025-07-24 11:41:34\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3431993\', \'106.7887784\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'11:41:34\', \'2025-07-24\', \'1753332093_52ec45cb5f721a1172bd.jpg\', \'2025-07-24 11:41:34\', \'2025-07-24 11:41:34\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (:user_id:, :latitude:, :longitude:, :address:, :shift:, :work_location_type:, :time_in:, :attendance_date:, :photo_in:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
CRITICAL - 2025-07-24 11:41:34 --> [Caused by] mysqli_sql_exception: Unknown column 'latitude' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3431993\', \'106.7887784\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'11:41:34\', \'2025-07-24\', \'1753332093_52ec45cb5f721a1172bd.jpg\', \'2025-07-24 11:41:34\', \'2025-07-24 11:41:34\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3431993\', \'106.7887784\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'11:41:34\', \'2025-07-24\', \'1753332093_52ec45cb5f721a1172bd.jpg\', \'2025-07-24 11:41:34\', \'2025-07-24 11:41:34\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3431993\', \'106.7887784\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'11:41:34\', \'2025-07-24\', \'1753332093_52ec45cb5f721a1172bd.jpg\', \'2025-07-24 11:41:34\', \'2025-07-24 11:41:34\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (:user_id:, :latitude:, :longitude:, :address:, :shift:, :work_location_type:, :time_in:, :attendance_date:, :photo_in:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
ERROR - 2025-07-24 11:42:51 --> mysqli_sql_exception: Unknown column 'latitude' in 'field list' in E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `at...', 0)
#1 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `at...')
#2 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `at...')
#3 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `at...', Array, false)
#4 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 E:\laragon\www\apici\vendor\codeigniter4\framework\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 E:\laragon\www\apici\app\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert(Array)
#8 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
#9 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
#10 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 E:\laragon\www\apici\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 E:\laragon\www\apici\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('E:\\laragon\\www\\...')
#15 {main}
CRITICAL - 2025-07-24 11:42:51 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'latitude' in 'field list'
[Method: POST, Route: api/attendance/checkin]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (:user_id:, :latitude:, :longitude:, :address:, :shift:, :work_location_type:, :time_in:, :attendance_date:, :photo_in:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
CRITICAL - 2025-07-24 11:42:51 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'latitude' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3431993\', \'106.7887784\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'11:42:51\', \'2025-07-24\', \'1753332171_28a68cbdd88a82922f78.jpg\', \'2025-07-24 11:42:51\', \'2025-07-24 11:42:51\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3431993\', \'106.7887784\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'11:42:51\', \'2025-07-24\', \'1753332171_28a68cbdd88a82922f78.jpg\', \'2025-07-24 11:42:51\', \'2025-07-24 11:42:51\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (:user_id:, :latitude:, :longitude:, :address:, :shift:, :work_location_type:, :time_in:, :attendance_date:, :photo_in:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
CRITICAL - 2025-07-24 11:42:51 --> [Caused by] mysqli_sql_exception: Unknown column 'latitude' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3431993\', \'106.7887784\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'11:42:51\', \'2025-07-24\', \'1753332171_28a68cbdd88a82922f78.jpg\', \'2025-07-24 11:42:51\', \'2025-07-24 11:42:51\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3431993\', \'106.7887784\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'11:42:51\', \'2025-07-24\', \'1753332171_28a68cbdd88a82922f78.jpg\', \'2025-07-24 11:42:51\', \'2025-07-24 11:42:51\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3431993\', \'106.7887784\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'11:42:51\', \'2025-07-24\', \'1753332171_28a68cbdd88a82922f78.jpg\', \'2025-07-24 11:42:51\', \'2025-07-24 11:42:51\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (:user_id:, :latitude:, :longitude:, :address:, :shift:, :work_location_type:, :time_in:, :attendance_date:, :photo_in:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
ERROR - 2025-07-24 11:46:02 --> mysqli_sql_exception: Unknown column 'latitude' in 'field list' in E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `at...', 0)
#1 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `at...')
#2 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `at...')
#3 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `at...', Array, false)
#4 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 E:\laragon\www\apici\vendor\codeigniter4\framework\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 E:\laragon\www\apici\app\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert(Array)
#8 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
#9 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
#10 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 E:\laragon\www\apici\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 E:\laragon\www\apici\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('E:\\laragon\\www\\...')
#15 {main}
CRITICAL - 2025-07-24 11:46:02 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'latitude' in 'field list'
[Method: POST, Route: api/attendance/checkin]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (:user_id:, :latitude:, :longitude:, :address:, :shift:, :work_location_type:, :time_in:, :attendance_date:, :photo_in:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
CRITICAL - 2025-07-24 11:46:02 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'latitude' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3432011\', \'106.7888036\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'11:46:02\', \'2025-07-24\', \'1753332362_db47733fff0ae016b26f.jpg\', \'2025-07-24 11:46:02\', \'2025-07-24 11:46:02\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3432011\', \'106.7888036\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'11:46:02\', \'2025-07-24\', \'1753332362_db47733fff0ae016b26f.jpg\', \'2025-07-24 11:46:02\', \'2025-07-24 11:46:02\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (:user_id:, :latitude:, :longitude:, :address:, :shift:, :work_location_type:, :time_in:, :attendance_date:, :photo_in:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
CRITICAL - 2025-07-24 11:46:02 --> [Caused by] mysqli_sql_exception: Unknown column 'latitude' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3432011\', \'106.7888036\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'11:46:02\', \'2025-07-24\', \'1753332362_db47733fff0ae016b26f.jpg\', \'2025-07-24 11:46:02\', \'2025-07-24 11:46:02\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3432011\', \'106.7888036\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'11:46:02\', \'2025-07-24\', \'1753332362_db47733fff0ae016b26f.jpg\', \'2025-07-24 11:46:02\', \'2025-07-24 11:46:02\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3432011\', \'106.7888036\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'11:46:02\', \'2025-07-24\', \'1753332362_db47733fff0ae016b26f.jpg\', \'2025-07-24 11:46:02\', \'2025-07-24 11:46:02\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (:user_id:, :latitude:, :longitude:, :address:, :shift:, :work_location_type:, :time_in:, :attendance_date:, :photo_in:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
ERROR - 2025-07-24 11:54:04 --> mysqli_sql_exception: Unknown column 'latitude' in 'field list' in E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `at...', 0)
#1 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `at...')
#2 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `at...')
#3 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `at...', Array, false)
#4 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 E:\laragon\www\apici\vendor\codeigniter4\framework\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 E:\laragon\www\apici\app\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert(Array)
#8 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
#9 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
#10 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 E:\laragon\www\apici\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 E:\laragon\www\apici\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('E:\\laragon\\www\\...')
#15 {main}
CRITICAL - 2025-07-24 11:54:04 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'latitude' in 'field list'
[Method: POST, Route: api/attendance/checkin]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (:user_id:, :latitude:, :longitude:, :address:, :shift:, :work_location_type:, :time_in:, :attendance_date:, :photo_in:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
CRITICAL - 2025-07-24 11:54:04 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'latitude' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3431955\', \'106.7887755\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'11:54:04\', \'2025-07-24\', \'1753332844_53a268c0babd6c77272a.jpg\', \'2025-07-24 11:54:04\', \'2025-07-24 11:54:04\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3431955\', \'106.7887755\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'11:54:04\', \'2025-07-24\', \'1753332844_53a268c0babd6c77272a.jpg\', \'2025-07-24 11:54:04\', \'2025-07-24 11:54:04\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (:user_id:, :latitude:, :longitude:, :address:, :shift:, :work_location_type:, :time_in:, :attendance_date:, :photo_in:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
CRITICAL - 2025-07-24 11:54:04 --> [Caused by] mysqli_sql_exception: Unknown column 'latitude' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3431955\', \'106.7887755\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'11:54:04\', \'2025-07-24\', \'1753332844_53a268c0babd6c77272a.jpg\', \'2025-07-24 11:54:04\', \'2025-07-24 11:54:04\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3431955\', \'106.7887755\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'11:54:04\', \'2025-07-24\', \'1753332844_53a268c0babd6c77272a.jpg\', \'2025-07-24 11:54:04\', \'2025-07-24 11:54:04\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3431955\', \'106.7887755\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'11:54:04\', \'2025-07-24\', \'1753332844_53a268c0babd6c77272a.jpg\', \'2025-07-24 11:54:04\', \'2025-07-24 11:54:04\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (:user_id:, :latitude:, :longitude:, :address:, :shift:, :work_location_type:, :time_in:, :attendance_date:, :photo_in:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
ERROR - 2025-07-24 12:03:34 --> mysqli_sql_exception: Unknown column 'latitude' in 'field list' in E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `at...', 0)
#1 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `at...')
#2 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `at...')
#3 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `at...', Array, false)
#4 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 E:\laragon\www\apici\vendor\codeigniter4\framework\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 E:\laragon\www\apici\app\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert(Array)
#8 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
#9 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
#10 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 E:\laragon\www\apici\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 E:\laragon\www\apici\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('E:\\laragon\\www\\...')
#15 {main}
CRITICAL - 2025-07-24 12:03:34 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'latitude' in 'field list'
[Method: POST, Route: api/attendance/checkin]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (:user_id:, :latitude:, :longitude:, :address:, :shift:, :work_location_type:, :time_in:, :attendance_date:, :photo_in:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
CRITICAL - 2025-07-24 12:03:34 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'latitude' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3431955\', \'106.7887755\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'12:03:34\', \'2025-07-24\', \'1753333414_615008df65a07cad7e05.jpg\', \'2025-07-24 12:03:34\', \'2025-07-24 12:03:34\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3431955\', \'106.7887755\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'12:03:34\', \'2025-07-24\', \'1753333414_615008df65a07cad7e05.jpg\', \'2025-07-24 12:03:34\', \'2025-07-24 12:03:34\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (:user_id:, :latitude:, :longitude:, :address:, :shift:, :work_location_type:, :time_in:, :attendance_date:, :photo_in:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
CRITICAL - 2025-07-24 12:03:34 --> [Caused by] mysqli_sql_exception: Unknown column 'latitude' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3431955\', \'106.7887755\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'12:03:34\', \'2025-07-24\', \'1753333414_615008df65a07cad7e05.jpg\', \'2025-07-24 12:03:34\', \'2025-07-24 12:03:34\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3431955\', \'106.7887755\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'12:03:34\', \'2025-07-24\', \'1753333414_615008df65a07cad7e05.jpg\', \'2025-07-24 12:03:34\', \'2025-07-24 12:03:34\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3431955\', \'106.7887755\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'12:03:34\', \'2025-07-24\', \'1753333414_615008df65a07cad7e05.jpg\', \'2025-07-24 12:03:34\', \'2025-07-24 12:03:34\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (:user_id:, :latitude:, :longitude:, :address:, :shift:, :work_location_type:, :time_in:, :attendance_date:, :photo_in:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
ERROR - 2025-07-24 12:03:42 --> mysqli_sql_exception: Unknown column 'latitude' in 'field list' in E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `at...', 0)
#1 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `at...')
#2 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `at...')
#3 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `at...', Array, false)
#4 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 E:\laragon\www\apici\vendor\codeigniter4\framework\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 E:\laragon\www\apici\app\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert(Array)
#8 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
#9 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
#10 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 E:\laragon\www\apici\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 E:\laragon\www\apici\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('E:\\laragon\\www\\...')
#15 {main}
CRITICAL - 2025-07-24 12:03:42 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'latitude' in 'field list'
[Method: POST, Route: api/attendance/checkin]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (:user_id:, :latitude:, :longitude:, :address:, :shift:, :work_location_type:, :time_in:, :attendance_date:, :photo_in:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
CRITICAL - 2025-07-24 12:03:42 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'latitude' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3432032\', \'106.7887838\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'12:03:42\', \'2025-07-24\', \'1753333422_60d7590278a0440d506c.jpg\', \'2025-07-24 12:03:42\', \'2025-07-24 12:03:42\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3432032\', \'106.7887838\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'12:03:42\', \'2025-07-24\', \'1753333422_60d7590278a0440d506c.jpg\', \'2025-07-24 12:03:42\', \'2025-07-24 12:03:42\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (:user_id:, :latitude:, :longitude:, :address:, :shift:, :work_location_type:, :time_in:, :attendance_date:, :photo_in:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
CRITICAL - 2025-07-24 12:03:42 --> [Caused by] mysqli_sql_exception: Unknown column 'latitude' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3432032\', \'106.7887838\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'12:03:42\', \'2025-07-24\', \'1753333422_60d7590278a0440d506c.jpg\', \'2025-07-24 12:03:42\', \'2025-07-24 12:03:42\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3432032\', \'106.7887838\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'12:03:42\', \'2025-07-24\', \'1753333422_60d7590278a0440d506c.jpg\', \'2025-07-24 12:03:42\', \'2025-07-24 12:03:42\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3432032\', \'106.7887838\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'12:03:42\', \'2025-07-24\', \'1753333422_60d7590278a0440d506c.jpg\', \'2025-07-24 12:03:42\', \'2025-07-24 12:03:42\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (:user_id:, :latitude:, :longitude:, :address:, :shift:, :work_location_type:, :time_in:, :attendance_date:, :photo_in:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
ERROR - 2025-07-24 12:04:18 --> mysqli_sql_exception: Unknown column 'latitude' in 'field list' in E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `at...', 0)
#1 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `at...')
#2 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `at...')
#3 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `at...', Array, false)
#4 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 E:\laragon\www\apici\vendor\codeigniter4\framework\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 E:\laragon\www\apici\app\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert(Array)
#8 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
#9 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
#10 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 E:\laragon\www\apici\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 E:\laragon\www\apici\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('E:\\laragon\\www\\...')
#15 {main}
CRITICAL - 2025-07-24 12:04:18 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'latitude' in 'field list'
[Method: POST, Route: api/attendance/checkin]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (:user_id:, :latitude:, :longitude:, :address:, :shift:, :work_location_type:, :time_in:, :attendance_date:, :photo_in:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
CRITICAL - 2025-07-24 12:04:18 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'latitude' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3432032\', \'106.7887838\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'12:04:18\', \'2025-07-24\', \'1753333458_a7b247a44ffc84808a99.jpg\', \'2025-07-24 12:04:18\', \'2025-07-24 12:04:18\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3432032\', \'106.7887838\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'12:04:18\', \'2025-07-24\', \'1753333458_a7b247a44ffc84808a99.jpg\', \'2025-07-24 12:04:18\', \'2025-07-24 12:04:18\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (:user_id:, :latitude:, :longitude:, :address:, :shift:, :work_location_type:, :time_in:, :attendance_date:, :photo_in:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
CRITICAL - 2025-07-24 12:04:19 --> [Caused by] mysqli_sql_exception: Unknown column 'latitude' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3432032\', \'106.7887838\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'12:04:18\', \'2025-07-24\', \'1753333458_a7b247a44ffc84808a99.jpg\', \'2025-07-24 12:04:18\', \'2025-07-24 12:04:18\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3432032\', \'106.7887838\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'12:04:18\', \'2025-07-24\', \'1753333458_a7b247a44ffc84808a99.jpg\', \'2025-07-24 12:04:18\', \'2025-07-24 12:04:18\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3432032\', \'106.7887838\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'12:04:18\', \'2025-07-24\', \'1753333458_a7b247a44ffc84808a99.jpg\', \'2025-07-24 12:04:18\', \'2025-07-24 12:04:18\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (:user_id:, :latitude:, :longitude:, :address:, :shift:, :work_location_type:, :time_in:, :attendance_date:, :photo_in:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
ERROR - 2025-07-24 12:09:28 --> mysqli_sql_exception: Unknown column 'latitude' in 'field list' in E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `at...', 0)
#1 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `at...')
#2 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `at...')
#3 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `at...', Array, false)
#4 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 E:\laragon\www\apici\vendor\codeigniter4\framework\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 E:\laragon\www\apici\app\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert(Array)
#8 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
#9 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
#10 E:\laragon\www\apici\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 E:\laragon\www\apici\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 E:\laragon\www\apici\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 E:\laragon\www\apici\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('E:\\laragon\\www\\...')
#15 {main}
CRITICAL - 2025-07-24 12:09:28 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'latitude' in 'field list'
[Method: POST, Route: api/attendance/checkin]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (:user_id:, :latitude:, :longitude:, :address:, :shift:, :work_location_type:, :time_in:, :attendance_date:, :photo_in:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
CRITICAL - 2025-07-24 12:09:28 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'latitude' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3432032\', \'106.7887838\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'12:09:28\', \'2025-07-24\', \'1753333768_af832252a78618d18afb.jpg\', \'2025-07-24 12:09:28\', \'2025-07-24 12:09:28\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3432032\', \'106.7887838\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'12:09:28\', \'2025-07-24\', \'1753333768_af832252a78618d18afb.jpg\', \'2025-07-24 12:09:28\', \'2025-07-24 12:09:28\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (:user_id:, :latitude:, :longitude:, :address:, :shift:, :work_location_type:, :time_in:, :attendance_date:, :photo_in:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
CRITICAL - 2025-07-24 12:09:28 --> [Caused by] mysqli_sql_exception: Unknown column 'latitude' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3432032\', \'106.7887838\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'12:09:28\', \'2025-07-24\', \'1753333768_af832252a78618d18afb.jpg\', \'2025-07-24 12:09:28\', \'2025-07-24 12:09:28\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3432032\', \'106.7887838\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'12:09:28\', \'2025-07-24\', \'1753333768_af832252a78618d18afb.jpg\', \'2025-07-24 12:09:28\', \'2025-07-24 12:09:28\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (\'1\', \'-6.3432032\', \'106.7887838\', \'MQ4Q+MC4, Gandul, Cinere\', \'Shift 1 (07:00 - 15:00)\', \'WFA\', \'12:09:28\', \'2025-07-24\', \'1753333768_af832252a78618d18afb.jpg\', \'2025-07-24 12:09:28\', \'2025-07-24 12:09:28\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `attendances` (`user_id`, `latitude`, `longitude`, `address`, `shift`, `work_location_type`, `time_in`, `attendance_date`, `photo_in`, `created_at`, `updated_at`) VALUES (:user_id:, :latitude:, :longitude:, :address:, :shift:, :work_location_type:, :time_in:, :attendance_date:, :photo_in:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\api\Attendance.php(63): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Api\Attendance->checkin()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Api\Attendance))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('E:\\laragon\\www\\apici\\public\\index.php')
ERROR - 2025-07-24 12:51:37 --> Received Check-in Data: {"user_id":"1","latitude":"-6.3431947","longitude":"106.7887774","address":"Jl. Damai No.100, Gandul, Kecamatan Cinere","shift":"Shift 1 (07:00 - 15:00)","work_location_type":"WFA"}
ERROR - 2025-07-24 12:54:24 --> Received Check-in Data: {"user_id":"1","latitude":"-6.3432005","longitude":"106.7887809","address":"MQ4Q+MC4, Gandul, Cinere","shift":"Shift 1 (07:00 - 15:00)","work_location_type":"WFA"}
